frontend/src/metabase/dashboard/components/DashCard/DashCard.tsx

```diff
@@ -316,11 +316,9 @@ function DashCardInner({
             onAddSeries={() => onAddSeries(dashcard)}
             onRemove={() => onRemove(dashcard)}
             onReplaceCard={() => onReplaceCard(dashcard)}
-            onUpdateVisualizationSettings={settings =>
-              onUpdateVisualizationSettings(dashcard.id, settings)
-            }
-            onReplaceAllVisualizationSettings={settings =>
-              onReplaceAllVisualizationSettings(dashcard.id, settings)
+            onUpdateVisualizationSettings={onUpdateVisualizationSettings}
```

Comment:
Should we also wrap `DashCardActionsPanel` with `memo` to prevent unnecessary rerenders? And similarly in other places

Reply:
https://github.com/metabase/metabase/pull/43431